# HG changeset patch
# User Sergi Almacellas Abellana <sergi@koolpi.com>
Always return a list on ModelSQL create

issue4822
review16351002

Index: trytond/trytond/model/modelsql.py
===================================================================

--- a/trytond/trytond/model/modelsql.py
+++ b/trytond/trytond/model/modelsql.py
@@ -406,7 +406,7 @@
         super(ModelSQL, cls).create(vlist)

         if cls.table_query():
-            return False
+            return []

         table = cls.__table__()
         modified_fields = set()

